src/main/java/zmaster587/advancedRocketry/backwardCompat/WavefrontObject.java

```diff
@@ -82,11 +82,11 @@ private void loadObjModel(InputStream inputStream) throws ModelFormatException
                 lineCount++;
                 currentLine = currentLine.replaceAll("\\s+", " ").trim();
 
-                if (currentLine.startsWith("#") || currentLine.length() == 0)
+                /*if (currentLine.startsWith("#") || currentLine.length() == 0)
```

Comment:
Why was this commented out?

Reply:
Got it, cool. Dunno why that was there then

Reply:
Probably more for the benefit of the original developer than for code flow.